tests/test_datasource.py

```diff
@@ -45,8 +45,8 @@ def test_do_clip(mocker):
     bounds = (1, 5, 2, 6)
     mocker.patch('subprocess.check_call')
     cmd = datasource.do_clip(path='/tmp', bounds=bounds, output='/out.tif')
-    assert cmd == 'make -C /tmp clip OUTPUT="/out.tif" PROJWIN="1 6 2 5"'
-    subprocess.check_call.assert_called_once_with(cmd, shell=True)
+    assert re.match(r'make -C /tmp clip OUTPUT="/out\.tif" PROJWIN="1 6 2 5" RUN_ID=".{32}"', cmd)
```

Comment:
No need to `re.match` just `cmd.startswith()` is enough. And wey to you changed `assert_called_once_with` with `assert_called_with`?